src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+cron.expression=0 0 * ? * *
```

Comment:
Why do we need this?

Reply:
We already have yml file right?

Reply:
removed it. Added a section 'scheduler' under 'consentmanager' for the cron property in application.yml.

in application-dev.yml - configured to every minute
application.yml - every hour

Reply:
added to application.yml under the section consentmanager.scheduler.